enterprise/metrics/src/main/java/org/neo4j/metrics/source/ClusterMetrics.java

```diff
@@ -27,17 +27,21 @@
 
 import org.neo4j.kernel.configuration.Config;
 import org.neo4j.kernel.ha.SlaveUpdatePuller;
+import org.neo4j.kernel.impl.annotations.Documented;
 import org.neo4j.kernel.lifecycle.LifecycleAdapter;
 import org.neo4j.kernel.monitoring.Monitors;
 import org.neo4j.metrics.MetricsSettings;
 
 import static com.codahale.metrics.MetricRegistry.name;
 
+@Documented( ".Cluster Metrics" )
 public class ClusterMetrics extends LifecycleAdapter
 {
     private static final String NAME_PREFIX = "neo4j.cluster";
-    private static final String SLAVE_PULL_UPDATES = name( NAME_PREFIX, "slave_pull_updates" );
-    private static final String SLAVE_PULL_UPDATE_UP_TO_TX = name( NAME_PREFIX, "slave_pull_update_up_to_tx" );
+    @Documented( "The total number of pull updates executed by this instance" )
```

Comment:
"The total number of _update pulls_ executed by this instance" reads better to me.